scripts/scripts.js

```diff
@@ -120,6 +120,36 @@ export function toClassName(name) {
     : '';
 }
 
+/**
+ * Replace icons with inline SVG and prefix with codeBasePath.
+ * @param {Element} element
+ */
+function replaceIcons(element) {
+  element.querySelectorAll('img.icon').forEach((img) => {
+    const span = document.createElement('span');
+    span.className = img.className;
+    img.replaceWith(span);
+  });
+}
+
+/**
+ * Replace icons with inline SVG and prefix with codeBasePath.
+ * @param {Element} element
+ */
+export async function decorateIcons(element) {
```

Comment:
```suggestion
export function decorateIcons(element) {
```

As is, the method is not `async`.

Reply:
What about the `await fetch()`?